src/sass/partials/global/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$font-stack:    HelveticaNeue, Helvetica, Arial, sans-serif;
+
+$white: #fff;
+
+// Breakpoints
+$small: 900px;
```

Comment:
Breakpoints should be based on content; where it no longer looks good or where the design breaks. No small, medium, large or mobile, tablet, desktop sizes.